src/mongocxx/collection.cpp

```diff
@@ -83,6 +83,12 @@ collection::collection(const database& database, bsoncxx::stdx::string_view coll
           database.name(), database._impl->client_impl, collection_name.data())) {
 }
 
+collection::collection(const database& database, bsoncxx::stdx::string_view collection_name,
```

Comment:
I'll change these here, and let's coordinate our merges to avoid a rebase nightmare